common/static/common/js/discussion/views/thread_response_edit_view.js

```diff
@@ -35,13 +35,17 @@
                 return this.$el.find(selector);
             };
 
-            ThreadResponseEditView.prototype.initialize = function() {
+            ThreadResponseEditView.prototype.initialize = function(options) {
+                this.options = options;
+                this.startHeader = options.startHeader;
```

Comment:
Nit: is it necessary to store this.startHeader given that you also store this.options and reference this.options.startHeader below?